test/unit/mdc-menu/mdc-menu.test.js

```diff
@@ -16,422 +16,404 @@
-  document.body.removeChild(anchor);
+test('adapter#elementContainsClass returns true if the class exists on the element', () => {
+  const {root, component} = setupTest();
+  const firstItem = root.querySelector('.mdc-list-item');
+  firstItem.classList.add('foo');
+  const containsFoo = component.getDefaultFoundation().adapter_.elementContainsClass(firstItem, 'foo');
+  assert.isTrue(containsFoo);
 });
 
-test('adapter#isRtl returns false for explicit LTR documents', () => {
-  const anchor = bel`<div dir="ltr" class="mdc-menu-anchor"></div>`;
-  const {root, component} = setupTest(true);
-  anchor.appendChild(root);
-  document.body.appendChild(anchor);
-  assert.isNotOk(component.getDefaultFoundation().adapter_.isRtl());
-  document.body.removeChild(anchor);
+test('adapter#elementContainsClass returns false if the class does not exist on the element', () => {
+  const {root, component} = setupTest();
+  const firstItem = root.querySelector('.mdc-list-item');
+  const containsFoo = component.getDefaultFoundation().adapter_.elementContainsClass(firstItem, 'foo');
+  assert.isFalse(containsFoo);
 });
 
-test('adapter#isRtl returns false for implicit LTR documents', () => {
-  const anchor = bel`<div class="mdc-menu-anchor"></div>`;
-  const {root, component} = setupTest(true);
-  anchor.appendChild(root);
-  document.body.appendChild(anchor);
-  assert.isNotOk(component.getDefaultFoundation().adapter_.isRtl());
-  document.body.removeChild(anchor);
+test('adapter#closeSurface returns false if the class does not exist on the element', () => {
+  const {component, menuSurface} = setupTestWithFakes();
+  component.getDefaultFoundation().adapter_.closeSurface();
+  td.verify(menuSurface.hide(), {times: 1});
 });
 
-test('adapter#setTransformOrigin sets the correct transform origin on the menu element', () => {
+test('adapter#getElementIndex returns the index value of an element in the list', () => {
   const {root, component} = setupTest();
-  // Write expected value and read canonical value from browser.
-  root.style.webkitTransformOrigin = root.style.transformOrigin = 'left top 10px';
-  const expected = root.style.getPropertyValue(`${getTransformPropertyName(window)}-origin`);
-  // Reset value.
-  root.style.webkitTransformOrigin = root.style.transformOrigin = '';
+  const firstItem = root.querySelector('.mdc-list-item');
+  const indexValue = component.getDefaultFoundation().adapter_.getElementIndex(firstItem);
+  assert.equal(0, indexValue);
+});
 
-  component.getDefaultFoundation().adapter_.setTransformOrigin('left top 10px');
-  assert.equal(root.style.getPropertyValue(`${getTransformPropertyName(window)}-origin`), expected);
+test('adapter#getElementIndex returns -1 if the element does not exist in the list', () => {
+  const {component} = setupTest();
+  const firstItem = document.createElement('li');
+  const indexValue = component.getDefaultFoundation().adapter_.getElementIndex(firstItem);
+  assert.equal(-1, indexValue);
```

Comment:
The `assert.equal` API specifies `actual` before `expected`. These tests will probably produce confusing messages if they fail.